gradle.properties

```diff
@@ -28,9 +28,9 @@ https.connectionTimeout=60000
 
 # dependency versions
 kotlinVersion=1.9.23
-kotlinCoroutinesVersion=1.8.0
+kotlinCoroutinesVersion=1.8.1
 kotlinLoggingVersion=3.0.5
-mockitoKotlinVersion=5.2.1
+mockitoKotlinVersion=5.4.0
 mockitoInlineVersion=5.2.0
 
 camundaVersion=7.21.0
```

Comment:
We should be updating camunda as well if we update spring boot:
https://docs.camunda.org/manual/7.21/user-guide/spring-boot-integration/version-compatibility/

Reply:
fixed